pcs_test/tools/bin_mock/pcmk/crm_resource_mock.py

```diff
@@ -33,7 +33,6 @@ def main():
     argv = sys.argv[1:]
     if not argv:
         raise AssertionError()
-
```

Comment:
Please undo this change. There is no code change so we don't want the file to be mentioned in the commit as changed.